### PR TITLE
a little cleanup

### PR DESCRIPTION
**Pull request by grosser**
@byroot 

removing a few unnecessary things + making rakefile setup bundler in case it was not yet there

### CODE REVIEW

File: bootscale.gemspec [hunk 1]
```diff
@@ -13,10 +10,7 @@ Gem::Specification.new do |spec|
   spec.description   = %q{Inspired by Aaron Patterson's talk on the subject}
   spec.homepage      = "https://github.com/byroot/bootscale"
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = 'exe'
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ['lib']
+  spec.files         = `git ls-files -z lib README.md`.split("\x0")
```
Comment (byroot) [inline bootscale.gemspec]: I would keep LICENSE.txt too I think.

File: bootscale.gemspec [hunk 2]
```diff
@@ -1,7 +1,4 @@
-# coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'bootscale/version'
+require './lib/bootscale/version'
```
Comment (byroot) [inline bootscale.gemspec]: `require_relative` maybe? We explicitly don't support anything under 1.9.3

**Comment by byroot** [comment]
A couple nitpicks, otherwise LGTM.

**Comment by grosser** [comment]
updated

**Pull request merged**